### PR TITLE
Include doc for figgy-orangelight communication.

### LINKED ISSUE #3104: Document the different ways orangelight communicates with figgy to retrieve the viewer.
I started drafting https://docs.google.com/document/d/1MiltehCL79gspKp_wd4k7Bqo3VXguV6aIN5Z5VoGK24/edit 

related issues:
https://github.com/pulibrary/bibdata/issues/505 Remove figgy ark cache?
https://github.com/pulibrary/bibdata/issues/1720 Figgy arks resolving to voyager ids instead of Alma ids.
https://github.com/pulibrary/bibdata/issues/1724 Indexing 856 content on records without anh holdings or portfolios.
https://github.com/pulibrary/bibdata/issues/770 Figgy ARK overrides shouldn't replace the 856 label with 'Digital content' if it's not a link.
https://github.com/pulibrary/bibdata/issues/1848 Update fragment to be viewer-container
https://github.com/pulibrary/orangelight/issues/2147 Default book/manuscript thumbnail misses the icon to navigate to the viewer.

### PR DESCRIPTION
**Pull request by christinach**
closes #3104

### CODE REVIEW

File: docs/orangelight-figgy.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+In the record page we build in the DOM an element with class 'document-thumbnail' where we also include the bibid in a data-attribute:
+```
+<div class="document-thumbnail" data-bib-id="99106565893506421">
+  <div class="default"></div>
+</div>
+```
+
+1. If the specific record id exists as a scanned resource in figgy:
+Orangelight uses the bibid from the data attribute and invokes an async graphql call against figgy.
+   * For better performance we filter the resource that has no FileSet.
```
Comment (sandbergja) [inline docs/orangelight-figgy.md:10]: Does this mean the graphql query includes resources with or without FileSets?

**Review by sandbergja — COMMENTED** [review]
Thanks for this documentation, @christinach !  Just a small question.

File: docs/orangelight-figgy.md (hunk 1 above)
Comment (christinach) [reply docs/orangelight-figgy.md:10]: @sandbergja yes. We filter the results in the FiggyManifestManager. https://github.com/pulibrary/orangelight/blob/201c272894c1582c432d7eb0fc19e99373db1b29/app/javascript/orangelight/figgy_manifest_manager.js#L119

**Comment by christinach** [comment]
I need to add information about CDL.

**Review by kevinreiss — APPROVED** [review]

**Comment by kevinreiss** [comment]
We can add documentation for CDL in second commit when it is ready. Will keep 3104 open

**Pull request merged**